internal/command/instance_policy_label.go

```diff
@@ -373,6 +341,8 @@ func (c *Commands) getDefaultLabelPolicy(ctx context.Context) (*domain.LabelPoli
 	return policy, nil
 }
 
+// prepareAddDefaultLabelPolicy adds a default label policy, if none exists prior, and activates it directly
+// this functions is only used on instance setup so the policy can be activated directly
```

Comment:
It's not a breaking change, as the only thing changes it the check if a label policy is existing before activating it. This check is irrelevant because the label policy is created at the same time and the function itself is not used anywhere as default policies only get added (not changed) at instance setup.